src/drivers/indexeddb.js

```diff
@@ -181,27 +181,164 @@
             }
         }
 
+        // Initialize a singleton container for all running localForages.
+        if (!globalObject.container) {
+            globalObject.container = {};
```

Comment:
My bad, you're totally right about `globalObject`! It's the first thing I did on this PR and I forgot to change it. I will attach this container locally in the IndexedDB driver, name it `dbContexts`, as you suggest, and update the PR.